pkg/network.go

```diff
@@ -211,6 +211,9 @@ const (
 	// QueueProxyCertKey is the config for the secret name, which stores certificates
 	// to serve the TLS traffic from activator to queue-proxy.
 	QueueProxyCertKey = "queue-proxy-cert-secret"
+
+	// SecretInformerLabelKey is used to specify a label selector for informers listing ingress secrets.
+	SecretInformerLabelKey = "certificate.networking.knative.dev"
```

Comment:
`SecretInformerLabelKey` is too generic. 

If from the net-certmanager PR if we end up using the certificate UID as the label then

```go
CertifcateUIDLabelKey = "networking.knative.dev/certficateUID"
````

Reply:
the label could live here: https://github.com/knative/networking/blob/main/pkg/apis/networking/register.go

Reply:
Could be yes will change to that np.